CMakeLists.txt

```diff
@@ -1369,6 +1373,9 @@ else()
     target_include_directories(open62541-plugins PRIVATE ${PROJECT_BINARY_DIR}/src_generated)
     target_compile_definitions(open62541-plugins PRIVATE -DUA_DYNAMIC_LINKING_EXPORT)
     set_target_properties(open62541-plugins PROPERTIES FOLDER "open62541/lib")
+    if(UA_ENABLE_COVERAGE)
+        add_coverage(open62541-plugins)
```

Comment:
These are not used in the tests.
Just enabling the test plugins ought to be enough.

Reply:
Removed.